src/CookieJar.php

```diff
@@ -310,7 +310,7 @@ public function merge($arg)
 
 		}, func_get_args());
 
-		$data = call_user_func_array('Arr::merge', $arguments);
+		$data = call_user_func_array(__NAMESPACE__.'\Arr::merge', $arguments);
```

Comment:
There is an issue here in that using the namespace locks the instance of the `Arr` class that is used, meaning it becomes impossible to override. Ideally there needs to be a way to inject a class name so custom `Arr` classes can be used.